PurchasesTests/SwiftSources/LocalReceiptParsing/LocalReceiptParserTests.swift

```diff
@@ -11,7 +11,35 @@ import XCTest
 @testable import Purchases
 
 class LocalReceiptParserTests: XCTestCase {
-    func testCanInitialize() {
-        expect { LocalReceiptParser() } .notTo(raiseException())
+    
+    func testParseReceiptWithCustomReceiptParser() {
+        let receiptData = sampleReceiptData()
+        
+        expect {
+            let receipt = try ReceiptParser().parse(from: receiptData)
+            print(receipt.description)
+            return receipt
+        } .notTo(throwError())
```

Comment:
this is clearly not testing anything yet, next up is working on testing everything thoroughly